sync/sync_head.go

```diff
@@ -238,7 +240,10 @@ func (s *Syncer[H]) verifyBifurcating(ctx context.Context, subjHead, networkHead
 	s.metrics.failedBifurcation(ctx, int64(networkHead.Height()), networkHead.Hash().String())
 	log.Warnw("sync: header validation against subjHead", "height", networkHead.Height(), "hash", networkHead.Hash().String())
```

Comment:
same here

we should also change it to `bifurcation failed` similar to metrics. Otherwise, it's confusing for the same reason metric was.